readme.md

```diff
@@ -37,6 +37,22 @@ python manage.py runserver
 
 If there are new changes to the database the runserver output will run you through the process of updating and running the migrations.
 
+#### Running RabbitMQ & Celery independently
+RabbitMQ & Celery have been purposefully implemented in a way that allows them to be used in any part of the project.
+Equally, this also allows them to be used interactively in the Django Python shell.
+1. Install RabbitMQ `sudo apt-get install rabbitmq-server`
+1. Run RabbitMQ `sudo systemctl enable rabbitmq-server`
+1. Run the celery worker `celery -A web worker --loglevel=info`
```

Comment:
These lines also appear in the previous section for Ubuntu based Linux setup. Can they now be removed from there, since RabbitMQ and Celery aren't essential for running the server?

Reply:
When we start using tasks, they will be needed as part of the system. I appreciate that it is duplicating, however they are needed.

Reply:
My mistake. I haven't installed them myself yet - I suppose I'd better do that.